packages/schematics/angular/application/files/other-standalone-files/app.component.spec.ts.template

```diff
@@ -0,0 +1,33 @@
+import { TestBed } from '@angular/core/testing';<% if (routing) { %>
+import { RouterTestingModule } from '@angular/router/testing';<% } %>
```

Comment:
nit: `RouterTestingModule` won't be needed in v16+. Instead, tests can use the same router configuration that they do in the application: `provideRouter`

Reply:
Getting rid of `RouterTestingModule` is fine, but it needs to be replaced by importing `RouterOutlet` not `provideRouter`, based upon the needs of the initial app component when routing is true;

Reply:
I don't think this is correct. `AppComponent` should have itself already imported `RouterOutlet`. If `AppComponent` is standalone, adding `RouterOutlet` to the `TestBed` module would not make it available to `AppComponent`.